### PR TITLE
redesigned the css and html to incorporate a retro arcade design

### PR DESCRIPTION
**Pull request by enrightcw**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -1,6 +1,9 @@
 .container{
     margin: auto;
-    font-family: 'Press Start 2P', sans-serif;
+    color:#ff0070;
+    /* box-shadow: 0 0 10px #4666ff, 0 0 10px #4666ff; */
+    text-shadow: 0 0 8px #ff0080, 0 0 10px #ff0080;
+    /* font-size: 80px; */
```
Comment (enrightcw) [inline assets/css/style.css:6]: Good!

File: assets/css/style.css [hunk 2]
```diff
@@ -9,26 +12,33 @@ body{
     flex-direction: column;
 }
 
+/* .card{
+    background-color: black;
+    color: #ff0070
+} */
```
Comment (enrightcw) [inline assets/css/style.css:18]: Good!

File: assets/css/style.css [hunk 3]
```diff
@@ -9,26 +12,33 @@ body{
...
 main{
     flex: 1 0 auto;
 }
 
 .brand-logo{
-    color: white !important; 
+    color: #5ff5fd !important;
```
Comment (enrightcw) [inline assets/css/style.css:24]: Good!

File: assets/css/style.css [hunk 4]
```diff
@@ -9,26 +12,33 @@ body{
...
 
 .brand-logo{
-    color: white !important; 
+    color: #5ff5fd !important; 
     font-size: 25px !important;
 }
 
 a{
-    color: white !important;
+    color: #5ff5fd !important;
```
Comment (enrightcw) [inline assets/css/style.css:29]: Good!

Comment (enrightcw) [inline assets/css/style.css:29]: Good!

File: assets/css/style.css [hunk 5]
```diff
@@ -9,26 +12,33 @@ body{
...
     font-size: 15px !important;
+    text-shadow: 0 0 8px #5ff5fd, 0 0 10px #5ff5fd;
```
Comment (enrightcw) [inline assets/css/style.css:31]: Good!

Comment (enrightcw) [inline assets/css/style.css:31]: Good!

**Review by Darkorin — APPROVED** [review]

**Pull request merged**